tests/test_common.py

```diff
@@ -419,26 +423,28 @@ def test_timeout_waiting(self):
         NOTE: we intentionally not use the subprocess_launch_server fixture here
               to let the probing timeout.
         """
-        probing_timeout = 3
+        probing_timeout = self.LAUNCH_DELAY * 2
```

Comment:
:eye:

Reply:
Ah sorry it should be // here.

Reply:
Updated in [4f0fb0e](https://github.com/tier4/ota-client/pull/219/commits/4f0fb0ecd5021dfaaf70b7c68d6d924f92a76d3f).